app/models/authorized-account.ts

```diff
@@ -31,12 +32,15 @@ export default class AuthorizedAccountModel extends Model {
     @attr('fixstring') readonly authUrl!: string; // Only returned when POSTing to /authorized-xyz-accounts
     @attr('boolean') readonly credentialsAvailable!: boolean;
 
+    @belongsTo('external-storage-service')
+    externalStorageService!: AsyncBelongsTo<ExternalStorageServiceModel> & ExternalStorageServiceModel;
+
+
```

Comment:
Let's not add this to the base model. We should put it back on the `AuthorizedStorageAccount` model.